docs/ROADMAP.md

```diff
@@ -18,10 +18,13 @@ To discuss the roadmap and the related initiatives, use the
   * Android Studio DCL support preview
   * Toolability demonstrations (IDE integration, mutations)
 
-## Subsequent Early Access Previews
 
-* 2024-H2
-* More EAPs towards the end of the year
-* More features
-* Addressing collected feedback
-* Further feedback from the community
+## Second Early Access Preview (EAP) - November 2024
+
+## Third Early Access Previes (EAP) - February 2025
```

Comment:
```suggestion
## Third Early Access Preview (EAP) - February 2025
```

Reply:
Yep, I have a fix for this typo locally already